<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gameWidget</class>
 <widget class="QWidget" name="gameWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>728</width>
    <height>544</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gobang_by_livingsu</string>
  </property>
  <widget class="QPushButton" name="btn_return">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>420</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>10</y>
     <width>91</width>
     <height>141</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>   白棋</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLCDNumber" name="lcd_write"/>
    </item>
    <item>
     <widget class="QLabel" name="lb_white_position">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>160</y>
     <width>91</width>
     <height>141</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>   黑棋 </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLCDNumber" name="lcd_black"/>
    </item>
    <item>
     <widget class="QLabel" name="lb_black_position">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="lb_timeuse">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>469</y>
     <width>221</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>ai计算耗时:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_nodeNum">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>470</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>ai叶结点数:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_eval">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>469</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>ai局面估分:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_restart">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>360</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>重新开始</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_regret">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>480</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>悔棋</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
